sigopt/objects.py

```diff
@@ -51,12 +51,28 @@ def __eq__(self, other):
     )
 
 
-class Assignments(ApiObject):
-  def get(self, key):
-    return self._body.get(key)
+class _DictWrapper(ApiObject, dict):
```

Comment:
If we make them mutable, users can just edit them in place and then PUT them back if they want to change them. What would the reasoning be for making them immutable?